ci-operator/step-registry/openshift/microshift/infra/iso-build/openshift-microshift-infra-iso-build-commands.sh

```diff
@@ -42,33 +42,43 @@ chmod +x /tmp/iso.sh
 #
 ci_clone_src
 
-# Attempt downloading MicroShift RPMs from brew.
+# Attempt downloading latest MicroShift RPMs from brew.
 # This requires VPN access, which is only enabled for the cache jobs.
 if [[ "${JOB_NAME}" =~ .*-cache.* ]] ; then
     # See BREW_RPM_SOURCE variable definition in test/bin/common.sh
     src_path="/go/src/github.com/openshift/microshift"
     out_path="${src_path}/_output/test-images/brew-rpms"
-    # If the current release supports brew RPM download, get the latest RPMs from
-    # brew to be included in the source repository archive
     pushd "${src_path}" &>/dev/null
-    if [ -e ./test/bin/manage_brew_rpms.sh ] && ./test/bin/manage_brew_rpms.sh -h | grep -q 'version_type' ; then
-        y_version="$(cut -d'.' -f2 "${src_path}/Makefile.version.$(uname -m).var")"
-        bash -x ./scripts/fetch_tools.sh brew
-        # Fail the entire job if there is no access to brew, as this could render an incomplete cache.
-        if ! bash -x ./test/bin/manage_brew_rpms.sh access; then
-            echo "ERROR: Brew Hub site is not accessible"
-            exit 1
-        fi
-        LATEST_RELEASE_TYPE="$(awk -F= '/^LATEST_RELEASE_TYPE=/ {gsub(/[[:space:]"]/, "", $2); print $2}' ./test/bin/common_versions.sh)"
-        if [[ -z "${LATEST_RELEASE_TYPE}" ]]; then
-            LATEST_RELEASE_TYPE="ec"
-        fi
-        # Warn on failures, as there are periods of time where a specific release type RPM for a release may not be available.
-        bash -x ./test/bin/manage_brew_rpms.sh download "4.${y_version}" "${out_path}" "${LATEST_RELEASE_TYPE}" || echo "WARNING: Failed to download ${LATEST_RELEASE_TYPE} RPMs for 4.${y_version}"
-        bash -x ./test/bin/manage_brew_rpms.sh download "4.${y_version}" "${out_path}" "nightly" || echo "WARNING: Failed to download nightly RPMs for 4.${y_version}"
-        bash -x ./test/bin/manage_brew_rpms.sh download "4.$((${y_version} - 1))" "${out_path}" "zstream" || echo "WARNING: Failed to download zstream RPMs for 4.$((${y_version} - 1))"
-        bash -x ./test/bin/manage_brew_rpms.sh download "4.$((${y_version} - 2))" "${out_path}" "zstream"
+
+    # Check if if manage_brew_rpms.sh script exists
+    if ! [ -e ./test/bin/manage_brew_rpms.sh ] ; then
```

Comment:
Just realized this is also an exit 0 condition. In those branches where the file does not exist we need to skip all this entirely. The previous approach was only doing it if the file existed and was ready to act based on `version_type` in help.
Can we do early exit 0 here too?